job_templates/run_command_-_ansible_default.erb

```diff
@@ -0,0 +1,21 @@
+<%#
+name: Run Command - Ansible Default
+job_category: Ansible Commands
+description_format: Run %{command}
+snippet: false
+template_inputs:
+- name: command
+  required: true
+  input_type: user
+  description: Command to run on the host
+  advanced: false
+provider_type: Ansible
+kind: job_template
+model: JobTemplate
+%>
+
+---
+- hosts: all
+  tasks:
+    - command: "{{ command }}"
```

Comment:
here I think it would be extremely useful to use <%= input... %> so you can see the command in job invocation page and in preview, the new job invocation UI will make it more obvious but with current UI, it would be a huge win I think